systemtest/src/main/java/io/strimzi/systemtest/resources/specific/HelmResource.java

```diff
@@ -144,12 +146,13 @@ private void clusterOperator(long operationTimeout, long reconciliationInterval)
      * Delete CO deployed via helm chart.
      */
     private void deleteClusterOperator() {
-        ResourceManager.helmClient().delete(HELM_RELEASE_NAME);
+        ResourceManager.helmClient().delete(namespaceInstallTo, HELM_RELEASE_NAME);
         DeploymentUtils.waitForDeploymentDeletion(ResourceManager.getCoDeploymentName());
+        // TODO not needed ?
```

Comment:
this can be removed or?

Reply:
I guess so, what about line after? I am not sure why it was added, because IIUC everything should be removed by uninstalling helm release